merlin/systems/triton/conversions.py

```diff
@@ -121,10 +121,15 @@ def dict_array_to_triton_request(model_name, dictarray, input_col_names, output_
     """
     input_tensors = []
 
-    for name, column in dictarray.items():
+    for name, column in tensor_table.items():
         if name in input_col_names:
-            col_tensor = _triton_tensor_from_array(name, column.values)
-            input_tensors.append(col_tensor)
+            if column.is_ragged:
+                values = _triton_tensor_from_array(f"{name}__values", column.values)
```

Comment:
I'm wondering if it would be helpful to have some TensorTable -> Triton Tensor conversion built into core somehow (not blocking this PR). Or perhaps some way to  extend core with registered conversons here or part of another package?

Reply:
Yeah, that all sounds good to me. Not sure exactly what it should look like yet either though